acceptance-tests/src/test/kotlin/com/chutneytesting/acceptance/tests/AuthorizationsAcceptanceTest.kt

```diff
@@ -0,0 +1,176 @@
+/*
+ * SPDX-FileCopyrightText: 2017-2024 Enedis
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package com.chutneytesting.acceptance.tests
+
+import com.chutneytesting.acceptance.common.jsonHeader
+import com.chutneytesting.kotlin.dsl.*
+
+val `Declare a new role with its authorizations` = Scenario(title = "Declare a new role with its authorizations") {
+  Given("A new role") {
+    ContextPutAction(
+        entries = mapOf(
+            "newRole" to "jsonPath('{\"name\": \"NEW_ROLE\", \"rights\":[\"SCENARIO_READ\", \"CAMPAIGN_READ\", \"SCENARIO_WRITE\"]}', \"$\")".spEL,
+        )
+    )
+  }
+  When("Add it to current roles and authorizations") {
+    Step("Ask for current roles") {
+      HttpGetAction(
+          target = "CHUTNEY_LOCAL",
+          uri = "/api/v1/authorizations",
+          validations = mapOf(statusValidation(200)),
+          outputs = mapOf("currentAuthorizations" to "body".spEL())
+      )
+    }
+    Step("Add the new role") {
+      HttpPostAction(
+          target = "CHUTNEY_LOCAL",
+          uri = "/api/v1/authorizations",
+          headers = jsonHeader(),
+          body = """
+          {
+              "roles": ${'$'}{#jsonSerialize(#jsonPath(#currentAuthorizations, '$.roles').appendElement(#newRole))},
```

Comment:
Is `jsonSerialize` mandatory here ? It is not just below.
Have we some example for the others json functions, ie. `jsonMerge`, `jsonSet` ?

Reply:
jsonSerialize serialize the json in string, why do you want to change it to jsonMerge or jsonSet ?